### ISSUE TITLE
Error on upgrade to sufia 6.4.0: Cannot define multiple 'included' blocks for a Concern

### ISSUE THREAD

**Issue by hackmastera** [projecthydra/sufia#1402]
See stack trace here:
https://gist.github.com/hackmastera/f57f4aed2d2325c4cda8

The error resolves if you delete the first line of this file: 
https://github.com/projecthydra/sufia/blob/bc9763fcc7fb6213d5a58b2f2166d747f4f5dea5/app/controllers/concerns/sufia/my_controller_behavior.rb#L1

@awead ?

PR to follow.

**Comment by awead** [comment]
@hackmastera yeah, I don't think we need that require anymore.

**Comment by jcoyne** [comment]
@hackmastera you can cherry-pick this commit into the 6.0 stable branch and it should resolve: https://github.com/projecthydra/sufia/commit/2e35fb34f804bae2be851a4c75cb6852172f8157

**Comment by hackmastera** [comment]
Should we release 6.4.1 once the backport is finished? Or would that require another backport, due to intervening commits?

**Comment by hackmastera** [comment]
@awead @mjgiarlo I hereby agitate for a point release. opinions? and would this mean we also need a 6.4.x branch?

**Comment by mjgiarlo** [comment]
@hackmastera I am :+1: on a point release, cut from the 6.0-stable branch. Maybe we should rename the branch `6.x-stable`?

**Comment by hackmastera** [comment]
@mjgiarlo I feel like if we're just going to cut a release directly from 6.0-stable it should be 6.5 since there is other stuff and then it's not just a bug fix. I think it'd be easier to release 6.4.1 (6.4.0 + this commit) because then we don't have to worry about release notes and whether anything needs an upgrade path, etc.

**Comment by awead** [comment]
:+1: to point release, :+1: to `6.x-stable`

**Comment by hackmastera** [comment]
I can't tell whether y'all are advocating for the same thing as me. Maybe I should just prep some branches. (knew i should've created a new issue...)

**Comment by hackmastera** [comment]
resolved in  #1412

**Comment by mjgiarlo** [comment]
I'm cool with that too, @hackmastera.
